addons/mail/static/src/components/message_reactions_summary/message_reactions_summary.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionsSummary" owl="1">
+        <div  class="o_MessageReactionsSummary">
+            <t t-if="messageReactionsSummary">
+                <t t-if="messageReactionsSummary.messageReactionsSummaryViews[0] and messageReactionsSummary.messageReactionsSummaryViews[0].highlightedReaction">
```

Comment:
Component should only use getter to view model of self.
Never ever read a relation field on any other record whose target is a view model.

The component should likely be renamed to `MessageReactionsSummaryView` and receive a `MessageReactionSummaryView` record local id as prop.